### ISSUE TITLE
Ni No Kuni [BLES01555] All game characters and objects are missing black borders

### ISSUE THREAD

**Issue by P1llus** [RPCS3/rpcs3#3576]
I tried to find a issue about this already, but did not find it anywhere on github.

It seems that gameobjects that is not a static object on the map itself (NPC's, Cars, your characters etc) is missing their black borders, making them blend in with the landscape itself.

Here is the Emulator screenshot:
![image](https://user-images.githubusercontent.com/8027539/31429559-abfcc89c-ae6e-11e7-9d56-2cd8bc3e3461.png)

And here is it from PS3:
![image](https://user-images.githubusercontent.com/8027539/31429577-bad24bc6-ae6e-11e7-969f-3f01761e33e3.png)

This happens not only in this area, but in every area i have seen.

For some reason it sometimes work in cutscenes that uses the in-game engine, but this is rare cases, as the problems normally persist there as well.

GPU:
Both OpenGL and Vulcan
Resolution 1920x1080 and 1280x720
Invalidate Cache Every Frame
Use GPU Texture Scaling
Framelimit Auto

CPU:
LLVM
ASMJIT
Load liblv2.sprx only
2 Preferred threads
SPU loop detection

RPCS3 v0.0.3-3-2e5f8c0 Alpha | HEAD

Spec:
7700k
GTX 1080
16GB RAM

I do not know how relevant the logs are in this case, but here they are:
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/1374788/RPCS3.log.gz)

**Comment by Asinin3** [comment]
Yeah, it's a known bug but there wasn't an issue on it before it's not a regression I believe it has never worked on any previous RPCS3 builds.

**Comment by kd-11** [comment]
Superseded by https://github.com/RPCS3/rpcs3/issues/4178
